R/readENERDATA.R

```diff
@@ -0,0 +1,105 @@
+#' ENERDATA
+#'
+#' Read in an csv file and convert it to a magpie object
+#' The data has information about electricity capacity per country and per year
```

Comment:
the dataset does not contain only capacity data, it contains several types of data. suggested change: "The dataset contains several energy-related data types from ENERDATA for various countries and years."

Reply:
ok